internal/controller/postgrescluster/pgbouncer_test.go

```diff
@@ -365,3 +371,87 @@ func TestReconcilePGBouncerDeployment(t *testing.T) {
 	})
 
 }
+
+func TestReconcilePGBouncerDisruptionBudget(t *testing.T) {
+	env, cc, config := setupTestEnv(t, ControllerName)
+	t.Cleanup(func() { teardownTestEnv(t, env) })
+
+	r := &Reconciler{}
+	ctx, cancel := setupManager(t, config, func(mgr manager.Manager) {
```

Comment:
🔧 This test is calling reconcile methods directly, so there should be no Manager. (Manager watches for PostgresCluster objects and calls Reconcile.)